pkg/services/ngalert/notifier/channels/discord.go

```diff
@@ -0,0 +1,124 @@
+package channels
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/grafana/grafana/pkg/bus"
+	"github.com/grafana/grafana/pkg/setting"
+
+	"github.com/grafana/grafana/pkg/components/simplejson"
+
+	gokit_log "github.com/go-kit/kit/log"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/alerting"
+	old_notifiers "github.com/grafana/grafana/pkg/services/alerting/notifiers"
+	"github.com/prometheus/alertmanager/notify"
+	"github.com/prometheus/alertmanager/template"
+	"github.com/prometheus/alertmanager/types"
```

Comment:
Regrouping them. Plus gofmt/goimports needs to be run on it to sort them out (I have not sorted them).
```suggestion
	"context"
	"encoding/json"
	"fmt"
	"strconv"
	"strings"

	gokit_log "github.com/go-kit/kit/log"
	"github.com/pkg/errors"
	"github.com/prometheus/alertmanager/notify"
	"github.com/prometheus/alertmanager/template"
	"github.com/prometheus/alertmanager/types"

	"github.com/grafana/grafana/pkg/bus"
	"github.com/grafana/grafana/pkg/setting"
	"github.com/grafana/grafana/pkg/components/simplejson"
	"github.com/grafana/grafana/pkg/infra/log"
	"github.com/grafana/grafana/pkg/models"
	"github.com/grafana/grafana/pkg/services/alerting"
	old_notifiers "github.com/grafana/grafana/pkg/services/alerting/notifiers"
```

Reply:
gofmt and goimports are run as a file watcher in my IDE on save so they should be sorted correctly

Reply:
The suggestion was more about grouping. The auto formatting will only sort within a group. We usually follow upto 3 groups - standard libraries, non grafana, grafana (which is the suggestion above) OR 2 groups, standard libraries and others. Currently they seem to be randomly grouped.

Reply:
Gotcha! Thanks for that clarification.